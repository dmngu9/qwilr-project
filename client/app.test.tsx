import * as React from 'react';
import { shallow } from 'enzyme';

import App from './app';

describe('test enzyme', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<App />);

        expect(wrapper).toMatchSnapshot();
    });
});