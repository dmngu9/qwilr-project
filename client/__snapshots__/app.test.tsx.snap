// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test enzyme should render correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      exact={true}
      path="/about"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/profile"
      render={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
